requirements.txt

```diff
@@ -1,4 +1,3 @@
-https://download.pytorch.org/whl/cu90/torch-1.0.0-cp36-cp36m-linux_x86_64.whl
+torch==1.0.0
```

Comment:
Does this need to be == or would >= be ok?

Reply:
With the current version of the open source code, this needs to be ==. We need to make a few changes to allow 1.4.0. Do you think doing that later on in a separate PR would be okay?

Reply:
Yes, ok to do this later, thanks! Please create an issue describing what needs to be changed to make the code compatible with PyTorch 1.4.0

Reply:
Done thanks!